command/ca/provisioner/provisioner.go

```diff
@@ -72,3 +88,304 @@ $ step ca provisioner remove max@smallstep.com --kid 1234 --ca-config ca.json
+		return list
+	}
+	for _, rem := range rems {
+		for i, elem := range list {
+			if elem == rem {
+				list[i] = list[len(list)-1]
+				list = list[:len(list)-1]
+				break
+			}
+		}
+	}
+	return list
+}
+
+var (
+	x509TemplateFlag = cli.StringFlag{
+		Name:  "x509-template",
+		Usage: `The x509 certificate template <file>, a JSON representation of the certificate to create.`,
+	}
+	x509TemplateDataFlag = cli.StringFlag{
+		Name:  "x509-template-data",
+		Usage: `The x509 certificate template data <file>, a JSON map of data that can be used by the certificate template.`,
+	}
+	sshTemplateFlag = cli.StringFlag{
+		Name:  "ssh-template",
+		Usage: `The x509 certificate template <file>, a JSON representation of the certificate to create.`,
+	}
+	sshTemplateDataFlag = cli.StringFlag{
+		Name:  "ssh-template-data",
+		Usage: `The ssh certificate template data <file>, a JSON map of data that can be used by the certificate template.`,
+	}
+	x509MinDurFlag = cli.StringFlag{
+		Name:  "x509-min-dur",
+		Usage: `The minimum <duration> for an x509 certificate generated by this provisioner.`,
+	}
+	x509MaxDurFlag = cli.StringFlag{
+		Name:  "x509-max-dur",
+		Usage: `The maximum <duration> for an x509 certificate generated by this provisioner.`,
+	}
+	x509DefaultDurFlag = cli.StringFlag{
+		Name:  "x509-default-dur",
+		Usage: `The default <duration> for an x509 certificate generated by this provisioner.`,
+	}
+	sshUserMinDurFlag = cli.StringFlag{
+		Name:  "ssh-user-min-dur",
+		Usage: `The minimum <duration> for an ssh user certificate generated by this provisioner.`,
+	}
+	sshUserMaxDurFlag = cli.StringFlag{
+		Name:  "ssh-user-max-dur",
+		Usage: `The maximum <duration> for an ssh user certificate generated by this provisioner.`,
+	}
+	sshUserDefaultDurFlag = cli.StringFlag{
+		Name:  "ssh-user-default-dur",
+		Usage: `The maximum <duration> for an ssh user certificate generated by this provisioner.`,
+	}
+	sshHostMinDurFlag = cli.StringFlag{
+		Name:  "ssh-host-min-dur",
+		Usage: `The minimum <duration> for an ssh host certificate generated by this provisioner.`,
+	}
+	sshHostMaxDurFlag = cli.StringFlag{
+		Name:  "ssh-host-max-dur",
+		Usage: `The maximum <duration> for an ssh host certificate generated by this provisioner.`,
+	}
+	sshHostDefaultDurFlag = cli.StringFlag{
+		Name:  "ssh-host-default-dur",
+		Usage: `The maximum <duration> for an ssh host certificate generated by this provisioner.`,
+	}
+	disableRenewalFlag = cli.BoolFlag{
+		Name:  "disable-renewal",
+		Usage: `Disable renewal for all certificates generated by this provisioner.`,
+	}
+	allowRenewalAfterExpiryFlag = cli.BoolFlag{
+		Name:  "allow-renewal-after-expiry",
+		Usage: `Allow renewals for expired certificates generated by this provisioner.`,
+	}
+	enableX509Flag = cli.BoolFlag{
+		Name:  "x509",
+		Usage: `Enable provisioning of x509 certificates.`,
+	}
+	enableSSHFlag = cli.BoolFlag{
+		Name:  "ssh",
+		Usage: `Enable provisioning of ssh certificates.`,
```

Comment:
Document that this is enabled by default, at least I think this is what you did, and say how to disable it `--ssh=false`